### PR TITLE
use anr that fix flags based on avago version

### PR DESCRIPTION
**Pull request by felipemadero**

### CODE REVIEW

File: pkg/subnet/local.go [hunk 1]
```diff
@@ -115,7 +113,7 @@ func (d *LocalDeployer) BackendStartedHere() bool {
 //   - waits completion of operation
 //   - show status
 func (d *LocalDeployer) doDeploy(chain string, chainGenesis []byte, genesisPath string) (ids.ID, ids.ID, error) {
-	avagoVersion, avalancheGoBinPath, pluginDir, err := d.SetupLocalEnv()
+	_, avalancheGoBinPath, pluginDir, err := d.SetupLocalEnv()
```
Comment (holisticode) [inline pkg/subnet/local.go:116]: These are actually the only places where `d.SetupLocalEnv` seems to be returning the version.

If it's not needed anymore, should we eliminate that return param?

Comment (holisticode) [reply pkg/subnet/local.go:116]: Why do we actually not need it anymore?

Comment (felipemadero) [reply pkg/subnet/local.go:116]: It was used so as the CLI changed the ANR call based on avago version. It is not needed anymore if ANR handles that.

Comment (felipemadero) [reply pkg/subnet/local.go:116]: Will remove it from all functions

**Review by dasconnor — APPROVED** [review]
LGTM

**Pull request merged**